packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -92,29 +99,27 @@ library TaikoData {
         bytes32 difficulty; // slot 2
         bytes32 blobHash; //or txListHash (if Blob not yet supported), // slot 3
         bytes32 extraData; // slot 4
-        address coinbase; // L2 coinbase, // slot 5
+        bytes32 depositsHash; // slot 5
+        address coinbase; // L2 coinbase, // slot 6
         uint64 id;
         uint32 gasLimit;
-        uint64 timestamp; // slot 6
+        uint64 timestamp; // slot 7
         uint64 l1Height;
         uint16 minTier;
         bool blobUsed;
-        TaikoData.EthDeposit[] depositsProcessed; // slot 7
```

Comment:
Removal of deposits make the metadata smaller for proveBlock() to use, and it will also simplified the way we hash the metadata.